scripts/shared/lib/clusters_kind

```diff
@@ -126,79 +126,6 @@ function deploy_cni() {
-    echo "IMAGE_LIST=${IMAGE_LIST[*]}"
-    for image in "${IMAGE_LIST[@]}"
-    do
-        IMAGE_FAILURE=false
-
-        # Check if image is already present, and if not, download it.
-        echo "Processing Image: $image"
-        if [ -z "$(docker images -q "$image")" ] ; then
-            echo "Image $image not found, downloading..."
-            if ! docker pull "$image"; then
-                echo "**** 'docker pull $image' failed. Manually run. ****"
-                IMAGE_FAILURE=true
-            fi
-        else
-            echo "Image $image already downloaded."
-        fi
-
-        if [ "${IMAGE_FAILURE}" == false ] ; then
-            LCL_REG_IMAGE_NAME="${image/weaveworks/localhost:5000}"
-            # Copy image to local registry if not there
-            if [ -z "$(docker images -q "${LCL_REG_IMAGE_NAME}")" ] ; then
-                echo "Image ${LCL_REG_IMAGE_NAME} not found, tagging and pushing ..."
-                if ! docker tag "$image" "${LCL_REG_IMAGE_NAME}"; then
-                    echo "'docker tag $image ${LCL_REG_IMAGE_NAME}' failed."
-                    IMAGE_FAILURE=true
-                else
-                    if ! docker push "${LCL_REG_IMAGE_NAME}"; then
-                        echo "'docker push ${LCL_REG_IMAGE_NAME}' failed."
-                        IMAGE_FAILURE=true
-                    fi
-                fi
-            else
-                echo "Image ${LCL_REG_IMAGE_NAME} already present."
-            fi
-        fi
-
-        if [ "${IMAGE_FAILURE}" == false ] ; then
-            # Update the YAML by replacing upstream image name with the local registry image name
-            WEAVE_YAML=$(echo "${WEAVE_YAML}" | \
-                    image=${image} \
-                    LCL_REG_IMAGE_NAME=${LCL_REG_IMAGE_NAME} \
-                    yq e 'with(.items[] | select(.kind == "DaemonSet")| .spec.template.spec.containers[].image | select(. == strenv(image));
-                            . = strenv(LCL_REG_IMAGE_NAME) | . style="single") |
-                          with(.items[] | select(.kind == "DaemonSet")| .spec.template.spec.initContainers[].image | select(. == strenv(image));
-                            . = strenv(LCL_REG_IMAGE_NAME) | . style="single")
-                    ' - )
-        fi
-    done
-
-    echo "${WEAVE_YAML}" | kubectl apply -f -
-    echo "Waiting for weave-net pods to be ready..."
-    with_retries 5 ensure_weave_pods
-    kubectl wait --for=condition=Ready pods -l name=weave-net -n kube-system --timeout="${TIMEOUT}"
-    echo "Waiting for core-dns deployment to be ready..."
-    kubectl -n kube-system rollout status deploy/coredns --timeout="${TIMEOUT}"
-}
-
-function ensure_weave_pods() {
-    if kubectl get pods -l name=weave-net -n kube-system | grep weave-net; then
-       return 0
-    fi
-
-    sleep 3
-    return 1
-}
-
 function deploy_ovn_cni(){
     echo "OVN CNI deployed."
 }
```

Comment:
Optionally I'd also delete this, and the logic that calls it, since weave/ovn were the only ones we "supported" and now that weave is gone theres no need to even try anything.

If and when we add back another CNI, we can always bring back the code..

WDYT?